src/tests/ov_helpers/ov_models/src/input_layer.cpp

```diff
@@ -20,9 +20,14 @@ std::shared_ptr<ov::Node> makeInputLayer(const element::Type& type,
         input = ngraph::builder::makeConstant<float>(type, shape, {}, true);
         break;
     }
-    case ov::test::utils::InputLayerType::PARAMETER:
+    case ov::test::utils::InputLayerType::PARAMETER: {
         input = std::make_shared<ov::op::v0::Parameter>(type, ov::Shape(shape));
         break;
+    }
+    case ov::test::utils::InputLayerType::NONE: {
```

Comment:
We should also handle this in etlwise SetUp function (secondaryInputType)

Reply:
if you mean this place: https://github.com/openvinotoolkit/openvino/pull/20474/files#diff-d05b7832aa92ef8fe711eaea36408c17d34aa1414adf20ec5dc4a6819882a8abR204, then it is handled, thanks.